### PR TITLE
Add description to key binds

### PR DESCRIPTION
**Pull request by Moerliy**
Added docs for PR [hyprwm/Hyprland#6338](https://github.com/hyprwm/Hyprland/pull/6358) and fixed two code blocks in the binds section that had no highlight.

### CODE REVIEW

**Comment by Moerliy** [comment]
Should be good now. Please no typos :pray:

File: pages/Configuring/Binds.md [hunk 1]
```diff
@@ -137,14 +137,31 @@ You can view your switches in `hyprctl devices`.
...
 # to switch between windows in a floating workspace
 bind = SUPER,Tab,cyclenext,          # change focus to another window
 bind = SUPER,Tab,bringactivetotop,   # bring it to the top
 ```
 
 The keybinds will be executed in the order they were created. (top to bottom)
 
+### Description
+
+You can describe you keybind with the description flag.
```
Comment (vaxerski) [inline pages/Configuring/Binds.md:150]: your

**Review by vaxerski — COMMENTED** [review]
you got a typo :trollface:

**Comment by Moerliy** [comment]
fade-emoji

**Comment by Moerliy** [comment]
had a little upsi with my formater, my bad.

File: pages/Configuring/Binds.md (hunk 1 above)
Comment (Moerliy) [reply pages/Configuring/Binds.md]: fixed in commit c6cd521d8e848bd3ae5978ab118696eca5ff7e22

**Review by vaxerski — APPROVED** [review]

**Pull request merged**